packages/compass-app-stores/src/stores/instance-store.ts

```diff
@@ -398,45 +388,11 @@ export function createInstanceStore({
     onOpenResultNamespace
   );
 
-  const openModifyView = voidify(
-    async ({ ns, sameTab }: { ns: string; sameTab: boolean }) => {
-      const coll = await fetchCollectionDetails(ns);
-      if (coll?.sourceId && coll?.pipeline) {
-        // `modify-view` is currently implemented in a way where we are basically
```

Comment:
Not needed anymore, modifying view is just opening a collection workspace with certain already existing initial attributes now